core/server/models/base/index.js

```diff
@@ -399,6 +399,31 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
 
         if (options.context.user || ghostBookshelf.Model.isExternalUser(options.context.user)) {
             return options.context.user;
+        } else if (options.context.integration) {
```

Comment:
@gargol IMO we can just merge this for now. `options.context.integration` is null for now anyway.
It just shows how we can quickly support v0.1 with announcing the write API v2.

Reply:
Yeah, this makes sense. The only thing wanted to point out is that we don't have `integration` defined anywhere yet because the old name `api_key_id` is used. Maybe naming it `api_key_id` and doing a rename once we work on `integration` would be better :thinking:

Reply:
Yeah we have to figure out naming later and connect everything together 👍 I'd ignore that for now? 🤔

Reply:
Think naming it `api_key_id` would make it easier to find in the future in case we choose a name different to `integration`?